src/main/java/seedu/address/model/module/schedule/Schedule.java

```diff
@@ -89,14 +113,20 @@ public static boolean isTimeValid(String startTime, String endTime) {
      * @return conflict or not
      */
     public boolean isConflictWith(Schedule newSchedule) {
-        if (newSchedule.startTime.compareTo(this.endTime) > 0
-                || newSchedule.endTime.compareTo(this.startTime) < 0) {
-            return false;
-        }
-        if (!newSchedule.weekday.equals(this.weekday)) {
+        if (newSchedule.startTime.compareTo(this.endTime) >= 0
+                || newSchedule.endTime.compareTo(this.startTime) <= 0) {
             return false;
         }
-        return true;
+        return newSchedule.weekday.equals(this.weekday);
+    }
+
+    /**
+     * Checks if the module code and class type of another schedule match the module code
+     * and class type of current schedule.
+     * @param other another schedule
```

Comment:
Should this be `other schedule` or `another schedule`?

Reply:
Yes. Thanks for pointing out!